pkg/vault/test/foundry/unit/VaultBufferUnit.t.sol

```diff
@@ -406,6 +414,46 @@ contract VaultBufferUnitTest is BaseVaultTest {
         );
     }
 
+    function testWrapExactOutAmountInLessThanMin() public {
+        uint256 rate = 10_000;
+        wDaiInitialized.mockRate(rate);
+
+        BufferWrapOrUnwrapParams memory params = BufferWrapOrUnwrapParams({
+            kind: SwapKind.EXACT_OUT,
+            direction: WrappingDirection.WRAP,
+            wrappedToken: IERC4626(address(wDaiInitialized)),
+            amountGivenRaw: (_minWrapAmount - 1) * rate,
+            limitRaw: UINT256_MAX
+        });
+
+        vault.forceUnlock();
+
+        vm.expectRevert(
+            abi.encodeWithSelector(IVaultErrors.WrapAmountTooSmall.selector, IERC4626(address(wDaiInitialized)))
```

Comment:
I don't see this error used anywhere else. Can we check it for exact in too for consistency?

Reply:
Do you mean transfer `error WrapAmountTooSmall` to IVault?